teacher.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Teacher < Person
+  attr_reader :specialization
+
+  def initialize(specialization, age, name, parent_permission)
+    super(age, name, parent_permission)
+    @specialization = specialization
+  end
+
+  def can_use_services?
+    of_age? || @parent_permission ? true : false
+  end
```

Comment:
- [ ] `can_use_services?` method should always returns true according to the requirements, kindly fix it👌

![image](https://user-images.githubusercontent.com/45231276/188867781-89affce2-930f-4533-9170-9fee2597d67f.png)